bluebottle/projects/models.py

```diff
@@ -515,6 +515,18 @@ def get_tweet(self, **kwargs):
     class Meta(BaseProject.Meta):
         ordering = ['title']
 
+    class Analytics:
+        type = 'project'
+        tags = {
+            'sub_type': 'project_type',
+            'status': 'status.name',
+            'status_slug': 'status.slug',
+            'theme': 'theme.name',
+            'theme_slug': 'theme.slug',
+            'location': 'location.name',
+            'country': 'location.country.name'
```

Comment:
If the platform doesn't use locations the country is set directly on the project (`project.country`).
Furthermore you might get a property error with `location.name` if there is no location set.